src/main/java/address/controller/StatusBarFooterController.java

```diff
@@ -55,8 +56,12 @@ private void syncSchedulerOnTimeOut() {
     public void initStatusBar() {
         this.syncStatusBar = new StatusBar();
         this.updaterStatusBar = new StatusBar();
-        this.updaterStatusBar.getLeftItems().add(saveLocText);
-
+        saveLocText.setTextAlignment(TextAlignment.LEFT);
+        Tooltip tp = new Tooltip();
+        tp.textProperty().bind(saveLocText.textProperty());
+        saveLocText.setTooltip(tp);
+        this.updaterStatusBar.getRightItems().add(saveLocText);
+        saveLocText.setVisible(false);
```

Comment:
Abstract this to a method? Setting up of tooltip here breaks the abstraction level.